test/csuite/wt2695_checksum/main.c

```diff
@@ -321,6 +324,17 @@ main(int argc, char *argv[])
         }
     }
 
+    /*
+     * "Strobed" misalignments - test every combo of size/misalignment up to 16B.
+     */
+    for (i = 0; i < 16; i++) {     /* Length. */
+        for (j = 0; j < 16; j++) { /* Misalignment. */
```

Comment:
I'd suggest changing the names of i & j to length and misalignment and then the comments are no longer required.

Reply:
Thanks, you're right - I was being a bit lazy. Fixed in 7b663ae.

Reply:
other people have also used i, j, k etc. (i.e. line 304 in the same file) is that bad? or is it ok in some circumstances

Reply:
Good question @adelinexchen . I don't think there's a single yes/no answer, it's more of a matter of taste. For this new code, `i` and `j` represent fairly specific things (length and misalignment), so I took Jeremy's suggestion.

If you just need a generic counter, I reckon `i`/`j`/etc are fine.

Reply:
@korteland Ah, thank you!!